rotkehlchen/tests/unit/decoders/test_extrafi.py

```diff
@@ -5,14 +5,14 @@
 from rotkehlchen.accounting.structures.balance import Balance
 from rotkehlchen.assets.asset import Asset
 from rotkehlchen.chain.evm.decoding.constants import CPT_GAS
-from rotkehlchen.chain.evm.types import string_to_evm_address
-from rotkehlchen.chain.optimism.modules.extrafi.decoder import (
+from rotkehlchen.chain.evm.decoding.extrafi.decoder import (
     CPT_EXTRAFI,
     EXTRAFI_FARMING_CONTRACT,
     EXTRAFI_POOL_CONTRACT,
     EXTRAFI_STAKING_CONTRACT,
     VOTE_ESCROW,
 )
+from rotkehlchen.chain.evm.types import string_to_evm_address
```

Comment:
There is no new decoder test? Add one for the new chain too please. Perhaps a farm one. Base is the new one right?